backend/app/api/endpoints/adapter/tasks.py

```
@@ -236,6 +237,38 @@ def get_task_skills(
     )
 
 
+@router.get("/{task_id}/execution-info", response_model=TaskExecutionInfo)
+def get_task_execution_info(
+    task_id: int = Depends(with_task_telemetry),
+    current_user: User = Depends(security.get_current_user_jwt_apikey_tasktoken),
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)